packages/fxa-auth-server/lib/server.js

```diff
@@ -393,6 +394,9 @@ async function create(log, error, config, routes, db, oauthdb, translator) {
   );
   server.auth.strategy('oauthServerJWT', 'fxa-oauthServerJWT');
```

Comment:
`oauthServerJWT` ought to be obsolete now and removed in phase 3, or now maybe...

Reply:
Seems fine to defer to phase 3, though could cause less churn in cloudops deployment if the environment variables used to support this are removed.